src/mpyl/utilities/docker/__init__.py

```diff
@@ -303,11 +306,24 @@ def build(
 
 def login(logger: Logger, registry_config: DockerRegistryConfig) -> None:
     logger.info(f"Logging in with user '{registry_config.user_name}'")
-    docker.login(
-        server=f"https://{registry_config.host_name}",
-        username=registry_config.user_name,
-        password=registry_config.password,
-    )
+    if registry_config.provider == "azure":
+        docker.login(
+            server=f"https://{registry_config.host_name}",
+            username=registry_config.user_name,
+            password=registry_config.password,
+        )
+    if registry_config.provider == "aws":
+        docker.login_ecr(
+            aws_access_key_id=registry_config.user_name,
+            aws_secret_access_key=registry_config.password,
+            region_name="eu-central-1",
+            registry=registry_config.host_name,
+        )
+    else:
+        logger.debug(
+            "Container registry unknown. Check the registry settings inside the mpyl_config"
+        )
+        sys.exit()
```

Comment:
Uh actually, I assume you want to raise a `ValueError` so that it gets picked up by the exception handling of mpyl. See `registry_for_project` as an example